.github/workflows/release-swift-toolchain-binary-sizes.yml

```diff
@@ -104,7 +104,7 @@ jobs:
           arch: ${{ matrix.arch }}
 
       - name: Run google/bloaty
-        uses: thebrowsercompany/gha-google-bloaty@8481ad244140a67ebca1fca5eddf444f3065ce8f
+        uses: thebrowsercompany/gha-google-bloaty@main
```

Comment:
From what I understand, the team generally pins the dependencies.

Reply:
Oops I didn't mean to resolve this question (stated as an observation 🥸)

Reply:
Ah, my mistake for missing this! I've got no problem with pinning here.  Will fix